mgmtsystem_audit/data/audit_automated_actions.xml

```diff
@@ -0,0 +1,74 @@
+<odoo>
```

Comment:
I've just noticed there are some warnings about this xml file. I'm guessing these actions are not working properly. Can you check this?
```
2018-10-29 21:30:15,941 165 INFO openerp_test odoo.modules.loading: loading mgmtsystem_audit/data/audit_automated_actions.xml
2018-10-29 21:30:15,948 165 WARNING openerp_test odoo.models: ir.actions.server.create() includes unknown fields: body_html, condition, email_to, subject
2018-10-29 21:30:15,971 165 WARNING openerp_test odoo.models: base.automation.create() includes unknown fields: server_action_ids
2018-10-29 21:30:16,003 165 WARNING openerp_test odoo.models: ir.actions.server.create() includes unknown fields: body_html, condition, email_to, subject
2018-10-29 21:30:16,029 165 WARNING openerp_test odoo.models: base.automation.create() includes unknown fields: server_action_ids
```